examples/nextjs-with-typescript/src/Link.tsx

```diff
@@ -1,22 +1,19 @@
 /* eslint-disable jsx-a11y/anchor-has-content */
-import React from 'react';
+import React, { Ref, FunctionComponent, ReactNode } from 'react';
```

Comment:
The codebase convention is to go for `import * as React from 'react';`. What about doing the same here?

Reply:
Fixed